Application/EdFi.Ods.Features/OpenApiMetadata/Factories/OpenApiMetadataPathsFactory.cs

```diff
@@ -265,6 +263,21 @@ private Operation CreatePutByIdOperation(OpenApiMetadataPathsResource openApiMet
                 parameters = CreatePutParameters(openApiMetadataResource),
                 responses = OpenApiMetadataDocumentHelper.GetWriteOperationResponses(HttpMethod.Put)
             };
+
+            if(openApiMetadataResource.Resource.Entity.Identifier.IsUpdatable)
```

Comment:
Per our coding standards, you need a space between the `if` and the opening `(`.

Reply:
Rather than inline this logic after the object initialization, please follow the example of how some of the other properties are initialized here. Refactor this logic into a separate method (i.e. `GetDescription`, passing the `openApiMetadataResource` as the argument) and call that in the initializer, as is done for the `parameters` property (for example).

Reply:
updated